### DISCUSSION TITLE
Runtime error

### DISCUSSION THREAD

**Discussion by XWLZ11** [zhihulite/Hydrogen#63] [Q&A]
更新了0.520版本后，只能在回答外面看内容，可以正常刷新，但点进回答里会报错，

Runtime error: Invalid method call. Invalid Parameters.
public void androidx.viewpager2.widget.ViewPager2.registerOnPageChangeCallback(androidx.viewpager2.widget.ViewPager2$OnPageChangeCallback)
public void androidx.viewpager2.widget.ViewPager2.registerOnPageChangeCallback(androidx.viewpager2.widget.ViewPager2$OnPageChangeCallback)
public void androidx.viewpager2.widget.ViewPager2.registerOnPageChangeCallback(androidx.viewpager2.widget.ViewPager2$OnPageChangeCallback)

stack traceback:
	[C]: in field 'registerOnPageChangeCallback'
	/data/user/0/com.zhihu.hydrogen.x/files/answer.lua:632: in main chunk~~~~

**Comment by Firct** [comment]
这个要去issue提问题
